cicecore/drivers/nuopc/cmeps/ice_prescribed_mod.F90

```diff
@@ -211,6 +207,11 @@ subroutine ice_prescribed_init(clock, mesh, rc)
           hin_max(1) = 999._dbl_kind
        end if
 
+#ifndef CESMCOUPLED
+       ! If need initial cice values for coupling
+       call ice_prescribed_run(idate, msec)
+#endif
```

Comment:
UFS needed this call ice_prescribed_run at the end of ice_prescribed_init. I don't know if it's a bug for CESM not to. Presumably, the CICE state should be initialized in init, but it may depend on the components' run sequence. We left the #ifndef CESMCOUPLED so as not to impact CESM @dabail10